src/data/markdown/translated-guides/en/02 Using k6/05 Options.md

```diff
@@ -507,7 +519,7 @@ Pass the real system [environment variables](/using-k6/environment-variables) to
 | --- | --------------------------- | ------------------ | ---------------------------------------------------------------------------------------------------- |
 | N/A | `--include-system-env-vars` | N/A                | `true` for `k6 run`, but `false` for all other commands to prevent inadvertent sensitive data leaks. |
 
-<CodeGroup labels={[ "Shell" ]} lineNumbers={[true]}>
+<CodeGroup labels={["Shell" ]} lineNumbers={[false]}>
```

Comment:
Not strictly part of the PR, but it seemed unnecessary to have a line number for this.